lib/src/functions/math.dart

```diff
@@ -317,7 +317,7 @@ num _fuzzyRoundIfZero(num number) {
   return number.isNegative ? -0.0 : 0;
 }
 
-/// Returns a [Callable] named [name] that transforms a number's value
+/// Returns a [Callable.new] named [name] that transforms a number's value
```

Comment:
This one should probably just be `[Callable]`, since it's just being used to describe the return type.